src/xcode/ENA/ENA/Source/Services/Event Registration/QRCodePlayground.swift

```diff
@@ -0,0 +1,40 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import Base32
+
+// Since this is a POC this enum exists. Content should be moved into model/service that will later be used.
+enum QRCodePlayground {
+	
+	static func generateQRCode(with string: String, size: CGSize = CGSize(width: 400, height: 400)) -> UIImage? {
+
+		/// Create data from string which will be feed into the CoreImage Filter
+		guard let data = string.data(using: .shiftJIS) else {
+			return nil
```

Comment:
We should throw more things… like Errors :) 
![love-is-in-the-air-nennt](https://user-images.githubusercontent.com/149976/110512170-43f6cf00-8105-11eb-915a-0a9638bbb7ab.jpg)

Reply:
Throwing now :)